### PR TITLE
Add Stepcounter

### PR DESCRIPTION
**Pull request by grugnoymeme**
# Application Submission

Add Stepcounte app, author 47LeCoste

# Extra Requirements 

- [ Describe if your application requires any extra hardware or software ]

# Author Checklist (Fill this out)

- [ ] I've read the [contribution guidelines](../blob/HEAD/documentation/Contributing.md) and my PR follows them
- [ ] I own the code I'm submitting or have code owner's permission to submit it
- [ ] I [have validated](../blob/HEAD/documentation/Contributing.md#validating-manifest) the manifest file(s) with `python3 tools/bundle.py --nolint applications/CATEGORY/APPID/manifest.yml bundle.zip`

# Reviewer Checklist (Don't fill this out)

- [ ] Bundle is valid
- [ ] There are no obvious issues with the source code
- [ ] I've ran this application and verified its functionality

### CODE REVIEW

**Review by hedger — CHANGES_REQUESTED** [review]
Invalid commit SHA, please update

**Comment by grugnoymeme** [comment]
> Invalid commit SHA, please update

where to find?

File: applications/GPIO/stepcounter/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/grugnoymeme/flipperzero-StepCounter-fap.git
+    commit_sha: 8e7d7d59420c36ba730c83ebac12e46004d7352d
```
Comment (hedger) [inline applications/GPIO/stepcounter/manifest.yml:5]: here

Comment (grugnoymeme) [reply applications/GPIO/stepcounter/manifest.yml:5]: yes that one is the commit i made to this repo... where to find the correct one?

Comment (hedger) [reply applications/GPIO/stepcounter/manifest.yml:5]: it should point to the commit in your repo, with the version you want to add to app catalog. Check out documentation in this repo for details

**Comment by grugnoymeme** [comment]
updated

**Review by hedger — DISMISSED** [review]
Check out build errors. Run validation script locally to test changes, as pointed in the PR checklist

**Comment by doomwastaken** [comment]
It currently looks up README.md in your source folder, you can check errors in attached CI jobs

**Comment by doomwastaken** [comment]
Have you tried using https://github.com/flipperdevices/flipper-application-catalog/blob/main/documentation/Contributing.md#technical-requirements script for validation locally? It does not accept relative folders, sadly.

**Comment by doomwastaken** [comment]
All works great!

**Review by doomwastaken — APPROVED** [review]

**Comment by grugnoymeme** [comment]
At last! Lol 😂🤣

**Pull request merged**